config/rbac/role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: manager-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: manager-role
+subjects:
+- kind: ServiceAccount
+  name: default
```

Comment:
So we are using default ServiceAccount for Kogito operator now? Wouldn't it cause some potential security risk in case of namespace scoped deployments?

Reply:
I am not sure but we would have to test how  and which SA OLM would use. IIRC in the CSV we don't define role_binding definition, just the permission and cluster permission required by the operator. This should only affect the yaml install and I believe we should push the OLM installation to users :). For the OLM I will deploy and see which SA is created and used :)

Reply:
I'll create a service_account.yaml with name as `kogito-operator` and give its reference in kustomization.yaml and role_binding.yaml . By default OLM gives the permstion default service account which as you mentioned is not good for security reasons. Thanks for pointing this :)

Reply:
This is not a problem as I see it. This account is created and managed within the namespace the operator is installed.

Reply:
So we keep it as it is with default SA?

Reply:
Yes

Reply:
Okay